src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -243,7 +244,16 @@ public static void CloseSarifLogs(IEnumerable<string> logFiles)
             foreach (int runIdToClear in runIdsToClear)
             {
                 CodeAnalysisResultManager.Instance.RunIndexToRunDataCache.Remove(runIdToClear);
-                SarifLocationTagger.RemoveAllTagsForRun(runIdToClear);
+            }
+
+            IComponentModel componentModel = (IComponentModel)Package.GetGlobalService(typeof(SComponentModel));
+            if (componentModel != null)
+            {
```

Comment:
Check for visual studio SDK for helper for this.

Reply:
Could not locate one. Did create helper though.

---
In reply to: [501945395](https://github.com/microsoft/sarif-visualstudio-extension/pull/214#discussion_r501945395) [](ancestors = 501945395)